builder/commands/update.py

```diff
@@ -38,17 +40,19 @@ def take_action(self, parsed_args):
             print parser.print_help()
             sys.exit(1)
         config = self.parse_config(parsed_args.conf)
-        yaml_file = os.path.join(parsed_args.yaml)
-
-        if os.path.isdir(yaml_file):
-            views = [view for view in os.listdir(yaml_file)]
-            for view_yaml in views:
-                self.log.debug("View file %s" % view_yaml)
-                self.read_update(config,
-                                 os.path.join(yaml_file + '/%s' % view_yaml))
-        else:
-            self.log.debug("View file %s" % yaml_file)
-            self.read_update(config, os.path.join(yaml_file))
+        for yaml_filename in parsed_args.yaml:
+
+            yaml_file = os.path.join(yaml_filename)
+
+            if os.path.isdir(yaml_file):
+                views = [view for view in os.listdir(yaml_file)]
```

Comment:
I thought more about it. I think it is good as it is, we just need to document it well. We should probably add a more detailed description to the yaml option in the cli args.